locales/en-US/editor.properties

```diff
@@ -116,6 +116,7 @@ CMD_FILE_NEW=New File
 CMD_FILE_NEW_FOLDER=New Folder
 CMD_FILE_RENAME=Rename
 CMD_FILE_DELETE=Delete
+CMD_FILE_DOWNLOAD=Download
```

Comment:
Talk to @gideonthomas about the dance we need to do for this string, since it also needs to get added to Thimble.